docs/layers/icon-layer.md

```diff
@@ -0,0 +1,66 @@
+# Icon Layer
+
+The Icon Layer renders raster icons at given latitude/longitude coordinates.
+
+    import {IconLayer} from 'deck.gl';
+
+Inherits from all [Base Layer](/docs/layers/base-layer.md) properties.
+
+## Layer-specific Properties
+
+##### `iconAtlas` (Texture2D | String, required)
+
+Atlas image url or texture
+
+##### `iconMapping` (Object, required)
+
+Icon names mapped to icon definitions. Each icon is defined with the following values:
+
+  - `x`: x position of icon on the atlas image
+
+  - `y`: y position of icon on the atlas image
+
+  - `width`: width of icon on the atlas image
+
+  - `height`: height of icon on the atlas image
+
+  - `mask`: whether icon is treated as a transparency mask.
+  If `true`, user defined color is applied.
+  If `false`, pixel color from the image is applied.
+  Default: `false`
```

Comment:
I'm removing `getOffset` and add `anchorX` and `anchorY` to the icon definition.